compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3095,8 +3095,14 @@ object Parsers {
           mods |= Param
         }
         atSpan(start, nameStart) {
-          val name = ident()
-          acceptColon()
+          val name =
+            val name0 = ident()
+            if name0 == nme.using && !in.isColon then
```

Comment:
I think the same logic should be applied `erased`. Like using, it is a soft modifier that can only appear at the start of a parameter clause.

Reply:
What's `erased` again? Oh, yeah! Refresher. Thanks for the quick review.

Reply:
Apparently erase syntax was updated last year.